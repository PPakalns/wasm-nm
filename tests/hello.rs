#![cfg(target_arch = "wasm32")]

#[no_mangle]
pub fn fluxions(x: usize) -> usize {
    unsafe { imported(x) }
}

#[no_mangle]
pub fn quicksilver(_: usize) {}

extern {
    fn imported(x: usize) -> usize;
}

fn main() {}
