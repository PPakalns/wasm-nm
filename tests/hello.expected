i imported
e memory
e fluxions
e quicksilver
e main
e __ashlti3
e __ashrti3
e __lshrti3
e __ashldi3
e __ashrdi3
e __lshrdi3
e __divsi3
e __divdi3
e __divti3
e __modsi3
e __moddi3
e __modti3
e __divmodsi4
e __divmoddi4
e __multi3
e __muloti4
e __muldi3
e __mulosi4
e __mulodi4
e __floatsisf
e __floatsidf
e __floatdidf
e __floattisf
e __floattidf
e __floatunsisf
e __floatunsidf
e __floatundidf
e __floatuntisf
e __floatuntidf
e __fixsfsi
e __fixsfdi
e __fixsfti
e __fixdfsi
e __fixdfdi
e __fixdfti
e __fixunssfsi
e __fixunssfdi
e __fixunssfti
e __fixunsdfsi
e __fixunsdfdi
e __fixunsdfti
e __mulsf3
e __muldf3
e __udivmodti4
e __udivsi3
e __umodsi3
e __udivmodsi4
e __udivdi3
e __udivmoddi4
e __umoddi3
e __udivti3
e __umodti3
e memcpy
e memmove
e memset
e memcmp
e __powisf2
e __powidf2
e __addsf3
e __adddf3
e __subsf3
e __subdf3
e __divsf3
e __divdf3
e rust_eh_personality
